desktop/cmp/panel/impl/dragger/DraggerModel.js

```diff
@@ -54,6 +54,12 @@ export class DraggerModel extends HoistModel {
         this.panelEl = dragger.parentElement;
         const {panelEl: panel, panelModel} = this;
 
+        /** Workaround to allow dragging over iframe, which is its own separate document and cannot
+         listen for events from main document. {@see onDragEnd} */
+        if (XH.isDesktop) {
+            document.getElementById('xh-root').classList.add('xh-disable-iframe-pointer-events');
+        }
+
         throwIf(
             !panel.nextElementSibling && !panel.previousElementSibling,
             'Resizable panel has no sibling panel against which to resize.'
```

Comment:
you want to only muck with the dom at the last possible moment -- when you know the gesture has really started, and in particular nothing has thrown (see line 63).  If this code throws on line 63 your iFrames will be hosed

Reply:
good point!